### PR TITLE
Updating sizing of heading image for Safari Only

### PR DESCRIPTION
**Pull request by javielico**
Updating css rule to the hero image doesn't look squashed on Safari for Mac OSX

[Screenshot of the issue on Safari](https://i.postimg.cc/vDFxVCVn/Screenshot-2021-04-05-at-18-39-14.png)

### CODE REVIEW

**Comment by Sancus** [comment]
@Alecaddd Does this make sense to you? It seems super weird to me that we would need specific code to make Safari display images properly.... I don't have a Mac to test with.

File: assets/less/base/sizing.less [hunk 1]
```diff
@@ -34,6 +34,13 @@ each(@min-heights, #(@k, @v) {
   }
 });
 
+@media screen and (-webkit-min-device-pixel-ratio:0) {
+	.platform-img {
+		width: 100%;
+		height: 100%;
+	}
+}
+
```
Comment (Alecaddd) [inline assets/less/base/sizing.less:43]: No need to add the width: 100%, only the height.
Also, please respect the current file indentation.

**Review by Alecaddd — CHANGES_REQUESTED** [review]
Ah yes, the beauty of webkit.

**Comment by javielico** [comment]
Changes actioned.

**Review by Alecaddd — APPROVED** [review]

**Pull request merged**